### PR TITLE
some issue templates

### PR DESCRIPTION
**Pull request by bruno-garcia**
This will affect the whole org so I expect some discussion before we do anything.
Let's get it started.

Also worth noting that the template takes effect in repos that don't have already a template. It's just a default.

### CODE REVIEW

**Comment by BYK** [comment]
I'm generally +1 to this. Adding @dcramer @mattrobenolt @mitsuhiko @benvinegar @HazAT for feedback.

**Comment by bruno-garcia** [comment]
@markstory @HazAT @fdrabek @marandaneto @kamilogorek @rhcarvalho

File: ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!--
+Please consider including the answers below when relevant:
+
+How do you use Sentry:
+Sentry SaaS (sentry.io) or Self hosted (open-source, which version?)
+
+What SDK and version?
+e.g: JavaScript 5.11.1, .NET 1.2.0
```
Comment (rhcarvalho) [inline ISSUE_TEMPLATE/BUG_REPORT.md]: This is good to have, though if an issue is opened in a `sentry-foo` repo, I'd assume the SDK to be "foo".

**Comment by benvinegar** [comment]
I don't personally have a lot to add. Seems reasonable. Just a heads up that as of 1/23/2020, only paid Sentry.io customers receive email support – so the suggestion might be misleading to some.

**Comment by mattrobenolt** [comment]
I’d be -1 because as you said, this affects every repository, and iirc this makes it a bit annoying for internal users (us) when we create our own issues and whatnot. On top of that, I’d argue well over half of our repos, this messaging doesn’t even apply. On our internal repos, etc. I think this will be annoying. Willing to try, but I’m voicing my concern now.

File: ISSUE_TEMPLATE/BUG_REPORT.md (hunk 1 above)
Comment (bruno-garcia) [reply ISSUE_TEMPLATE/BUG_REPORT.md]: Right. I tried to use two lines and little text but this means. SDK name **and version**. But sure it's where applicable. As @mattrobenolt said, on our internal repos and other non-SDKs that'll be useless.

**Comment by bruno-garcia** [comment]
Worth pointing out it's just comments, to send a message to the author.

> Just a heads up that as of 1/23/2020, only paid Sentry.io customers receive email support – so the suggestion might be misleading to some.

What would be a better way to address that? I tried with _using sentry.io?_, is that accurate or depends on the plan? 

Also to consider is: Why would a user use our support if they can reach out to the developer directly through GitHub issue? I was hoping to have the wording here in a way that would incentivize paying customers to reach out to support instead of opening the issue altogether. Maybe point support has SLA?

>I’d be -1 because as you said, this affects every repository, and iirc this makes it a bit annoying for internal users (us) when we create our own issues and whatnot. 

I agree it'll be a bit of noise. In some repos neither Sentry version nor SDK are relevant.
But do we open issues on our internal repos a lot? As opposed to using JIRA, Asana or else? 
Please note there's no PR template in this PR.

Also, if one is actively developing on a repo, would it be OK to push an empty template on those repos (opt-out) then?

> On top of that, I’d argue well over half of our repos, this messaging doesn’t even apply. On our internal repos, etc. I think this will be annoying. Willing to try, but I’m voicing my concern now.

Again I agree. We could rollback this change if it turns out to be more trouble than not?

Btw, note that this will add an extra step in between clicking on _create issue_ and writing the text, like:

image

Another option is to simply add it to each repo where this is relevant.

** We could prepare an "SDK set of templates" and handle that within the team.

**Comment by mattrobenolt** [comment]
> But do we open issues on our internal repos a lot?

I’m sure this depends heavily on the team or project, but I do it quite a bit personally on some of our projects.

**Comment by untitaker** [comment]
Even if we are sure that this adds significant friction to issues we raise on our own repos, I am in favor of trying this out. There are more external people that raise more issues, so prompting them to include important information upfront will likely save us more time than we lose.

File: ISSUE_TEMPLATE/QUESTION.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!--
+You can have your question reach the wider Sentry community via:
+
+Stackoverflow.com Use the Sentry tag:
+https://stackoverflow.com/questions/ask
+
+Sentry forum is also a great knowledge base:
+https://forum.sentry.io/
+
```
Comment (marandaneto) [inline ISSUE_TEMPLATE/QUESTION.md]: what's about adding Discord too?

Comment (bruno-garcia) [reply ISSUE_TEMPLATE/QUESTION.md]: good catch

File: ISSUE_TEMPLATE/QUESTION.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Chat online with other Sentry users:
+https://discord.gg/Ww9hbqr
+
+For paying customers of sentry.io, there's also technical support:
```
Comment (bruno-garcia) [inline ISSUE_TEMPLATE/QUESTION.md]: @benvinegar tried to address your comment by writing this instead ^

File: ISSUE_TEMPLATE/BUG_REPORT.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<!--
+Please consider including the answers below when relevant:
+
+How do you use Sentry:
+Sentry SaaS (sentry.io) or Self hosted (open-source, which version?)
```
Comment (BYK) [inline ISSUE_TEMPLATE/BUG_REPORT.md]:
```suggestion
Sentry SaaS (sentry.io) or self-hosted/on-premise (open-source, which version?)
```

File: ISSUE_TEMPLATE/BUG_REPORT.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+What SDK and version?
```
Comment (BYK) [inline ISSUE_TEMPLATE/BUG_REPORT.md]:
```suggestion
Which SDK and version?
```

File: ISSUE_TEMPLATE/FEATURE_REQUEST.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<!--
+Please provide some context on what problem you are having and how this feature is going to solve it.
```
Comment (BYK) [inline ISSUE_TEMPLATE/FEATURE_REQUEST.md]:
```suggestion
Please provide some context on what problem you are trying to solve and how this feature is going to help.

```

File: ISSUE_TEMPLATE/QUESTION.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<!--
+You can have your question reach the wider Sentry community via:
+
+Stackoverflow.com Use the Sentry tag:
```
Comment (BYK) [inline ISSUE_TEMPLATE/QUESTION.md]:
```suggestion
StackOverflow.com with the Sentry tag:
```

**Comment by dashed** [comment]
@bruno-garcia thanks 👍

**Pull request merged**